components/collapse/nz-collapse-panel.component.html

```diff
@@ -5,7 +5,7 @@
     </ng-container>
   </ng-container>
   <ng-container *nzStringTemplateOutlet="nzHeader">{{ nzHeader }}</ng-container>
-  <div class="ant-collapse-extra" *ngIf="nzExtra">
+  <div class="ant-collapse-extra" *ngIf="nzExtra" (click)="$event.stopPropagation()">
```

Comment:
it's sucks. shouldn't we reorganize dom tree? if some guys using a tracking system based on event bubbling, dose it works for these kind tracking systems?

Reply:
Better idea?